boards/arm/nucleo_l552ze_q/nucleo_l552ze_q-common.dtsi

```diff
@@ -32,3 +32,36 @@
 		};
 	};
 };
+
+&flash0 {
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x00020000>;
+			read-only;
+		};
+		/* Secure image primary slot */
+		slot0_partition: partition@00011000 {
```

Comment:
It looks like there is an overlap with boot_partition which end at 0x00020000

Reply:
Good catch, fixed